src/datatable.js

```diff
@@ -137,18 +138,26 @@ export class DataTable {
 
   checkDisabled(action, row) {
     if (typeof action.disabled === 'function') {
+      this.hasVisibleActions = true;
```

Comment:
that might get problems with the execution order of checkDisabled and checkVisibility. anyways, might not be needed, pending maybe next comment